src/Simplex/Chat/Protocol.hs

```diff
@@ -223,6 +223,7 @@ data ChatMsgEvent (e :: MsgEncoding) where
   XGrpLeave :: ChatMsgEvent 'Json
   XGrpDel :: ChatMsgEvent 'Json
   XGrpInfo :: GroupProfile -> ChatMsgEvent 'Json
+  XGrpDirectInv :: ConnReqInvitation -> Maybe MsgContent -> ChatMsgEvent 'Json
```

Comment:
how would it work when it is Nothing? simply show an empty chat with another member in the list? do we want it?